src/Adapter/Tax/TaxComputer.php

```diff
@@ -52,14 +53,14 @@ public function __construct(
     /**
      * @param DecimalNumber $priceTaxExcluded
      * @param TaxRulesGroupId $taxRulesGroupId
-     * @param CountryId $countryId
+     * @param CountryIdInterface $countryId
      *
      * @return DecimalNumber
      */
     public function computePriceWithTaxes(
         DecimalNumber $priceTaxExcluded,
         TaxRulesGroupId $taxRulesGroupId,
-        CountryId $countryId
+        CountryIdInterface $countryId
```

Comment:
```suggestion
        CountryId $countryId
```

Reply:
Still needed I think, my previous comments disappeared

Reply:
yes it makes sens to expect only a CountryId . Applied